src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  --main-color: hsl(193, 100%, 43%);
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 1.2em;
+}
+
+li {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.moyo-img {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  align-self: center;
+}
+
+.nav-list {
+  display: flex;
+}
+
+.nav-list a {
```

Comment:
it's bad practice to increase specificity and add styles for tags. We can avoid it here. just add come class for your links